engines/kingdom/GPLogic1.cpp

```diff
@@ -0,0 +1,1501 @@
+/* ScummVM - Graphic Adventure Engine
```

Comment:
It's more standard to underscores in filenames rather than camel-case, though I'm not sure if this is fine with regard to the project's coding conventions

Reply:
you're right, I renamed to files to logic* (it wasn't able to notice a change of uppercase, maybe because it's not case sensitive under Windows?)